src/reedline_config.rs

```diff
@@ -168,17 +136,6 @@ pub(crate) fn create_keybindings(config: &Config) -> Result<KeybindingsMode, She
     match config.edit_mode.as_str() {
         "emacs" => {
             let mut keybindings = default_emacs_keybindings();
-            add_menu_keybindings(&mut keybindings);
-
-            // temporal keybinding with multiple events
-            keybindings.add_binding(
-                KeyModifiers::SHIFT,
-                KeyCode::BackTab,
-                ReedlineEvent::Multiple(vec![
-                    ReedlineEvent::Edit(vec![EditCommand::InsertChar('p')]),
```

Comment:
Am I losing my quick shell change? That's probably okay, we'll have enough examples folks can make one

Reply:
Oohhh, was that yours? Do you want me to add it back? You can create a keybinding for it

Reply:
I'll just add it to my personal one. It's not hard for other folks to add stuff like that